api/src/main/resources/applicationContext-service.xml

```diff
@@ -242,52 +135,6 @@
-	<bean id="roleValidator" class="org.openmrs.validator.RoleValidator"/>
-	<bean id="privilegeValidator" class="org.openmrs.validator.PrivilegeValidator"/>
-	<bean id="encounterTypeValidator" class="org.openmrs.validator.EncounterTypeValidator"/>
-	<bean id="conceptClassValidator" class="org.openmrs.validator.ConceptClassValidator"/>
-	<bean id="conceptDatatypeValidator" class="org.openmrs.validator.ConceptDatatypeValidator"/>
-	<bean id="conceptSourceValidator" class="org.openmrs.validator.ConceptSourceValidator"/>
-	<bean id="formEditValidator" class="org.openmrs.validator.FormValidator"/>
-	<bean id="fieldTypeValidator" class="org.openmrs.validator.FieldTypeValidator"/>
-	<bean id="programValidator" class="org.openmrs.validator.ProgramValidator"/>
-	<bean id="stateConversionValidator" class="org.openmrs.validator.StateConversionValidator"/>
-	<bean id="taskValidator" class="org.openmrs.validator.SchedulerFormValidator"/>
-	<bean id="obsValidator" class="org.openmrs.validator.ObsValidator"/>
-	<bean id="orderValidator" class="org.openmrs.validator.OrderValidator"/>
-	<bean id="drugOrderValidator" class="org.openmrs.validator.DrugOrderValidator"/>
-	<bean id="requireNameValidator" class="org.openmrs.validator.RequireNameValidator"/>
-	<bean id="conceptDrugValidator" class="org.openmrs.validator.ConceptDrugValidator"/>
-	<bean id="hl7SourceValidator" class="org.openmrs.validator.HL7SourceValidator"/>
-	<bean id="personAddressValidator" class="org.openmrs.validator.PersonAddressValidator"/>
-	<bean id="visitTypeValidator" class="org.openmrs.validator.VisitTypeValidator"/>
-	<bean id="visitAttributeTypeValidator" class="org.openmrs.validator.VisitAttributeTypeValidator"/>
-	<bean id="providerAttributeTypeValidator" class="org.openmrs.validator.ProviderAttributeTypeValidator"/>
-	<bean id="encounterValidator" class="org.openmrs.validator.EncounterValidator"/>
-	<bean id="locationAttributeTypeValidator" class="org.openmrs.validator.LocationAttributeTypeValidator"/>
-	<bean id="providerValidator" class="org.openmrs.validator.ProviderValidator"/>
-	<bean id="patientProgramValidator" class="org.openmrs.validator.PatientProgramValidator"/>
-	<bean id="conceptValidator" class="org.openmrs.validator.ConceptValidator"/>
-	<bean id="personMergeLogValidator" class="org.openmrs.validator.PersonMergeLogValidator"/>
-	<bean id="conceptReferenceTermValidator" class="org.openmrs.validator.ConceptReferenceTermValidator"/>
-	<bean id="conceptMapTypeValidator" class="org.openmrs.validator.ConceptMapTypeValidator"/>
-	<bean id="conceptAttributeTypeValidator" class="org.openmrs.validator.ConceptAttributeTypeValidator"/>
-	<bean id="implementationIdHttpClient" class="org.openmrs.util.HttpClient">
-		<constructor-arg name="url" value="https://implementation.openmrs.org"/>
-	</bean>
-
 	<!--  ************************ END VALIDATOR CONFIGURATION  ************************  -->
 
 	<context:component-scan base-package="org.openmrs">
```

Comment:
It should not be needed if you added it to the configuration class. You just need to point Spring to the OpenmrsApplicationContextConfig in addition to xml files.